src/frontend/web_application/src/components/form/DatePickerGroup/presenter.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker'; // https://github.com/Hacker0x01/react-datepicker
```

Comment:
`react-datepicker` being ~75 kB minified, it's a good candidate for async loading